txdbus/authentication.py

```diff
@@ -43,7 +44,9 @@ def beginAuthentication(self, protocol):
         
 
     def handleAuthMessage(self, line):
-        if not b' ' in line:
+        if not isinstance(line, six.binary_type):
+            line = six.b(line)
```

Comment:
Ah, thanks for pointing that out. I'll read the whole spec again before declaring this PR finished just in case.